pkg/occlient/occlient.go

```diff
@@ -1056,32 +1056,50 @@ func (c *Client) FollowBuildLog(buildName string) error {
 // kind is always required (can be set to 'all')
```

Comment:
You have removed `kind` and `name` parameters, you should also update comment!

Reply:
gotcha